editor/block-mover/generate-order-title.js

```diff
@@ -0,0 +1,29 @@
+export default function( { typeTitle, position, isFirst, isLast, dir } ) {
+	if ( isFirst && isLast ) {
+		return `Block "${ typeTitle }" is the only block, and cannot be moved`;
```

Comment:
To your question on translation, the [`i18n` utility](https://github.com/WordPress/gutenberg/tree/master/i18n) includes an sprintf implementation for string format replacement. To translate, this could be written as:

```js
/**
 * WordPress dependencies
 */
import { __, sprintf } from 'i18n';

// ...

return sprintf( __( 'Block "%s" is the only block, and cannot be moved' ), typeTitle );
```